2024Robot_Team3484/src/main/cpp/commands/teleop/TeleopIntakeCommand.cpp

```diff
@@ -0,0 +1,45 @@
+#include "commands/teleop/TeleopIntakeCommand.h"
+#include <Constants.h>
+
+using namespace IntakeConstants;
+
+TeleopIntakeCommand::TeleopIntakeCommand(IntakeSubsystem* intake_subsystem, OI*, oi)
+    : _intake_subsystem{intake_subsystem}, _oi{oi} {
+        AddRequirements(_intake_subsystem);
+}
+
+void TeleopIntakeCommand::Initialize() {}
+
+void TeleopIntakeCommand::Execute() {
+    if (_oi->__) { // TODO: Set button
+        if (!_intake_subsystem->HasPiece() || _oi->__) { // TODO: Set button
+            _intake_subsystem->SetIntakeAngle(IntakeConstants::INTAKE_POSITION);
+            _intake_subsystem->SetRollerPower(1);
```

Comment:
I suggest declaring changing the value passed into `SetRollerPower( )` to a constant (declared in `Constants.h`) and reference that value here. This will allow us to easily adjust the value and only have to change it in one place (especially if this value is the same for both auto and teleop commands).